doc/setup.md

```diff
@@ -0,0 +1,130 @@
+GeoCruncher developement setup
+===============================
+This section describes the setup necessary to get started with using geoCruncher.
+
+## Version control
+
+The source code for GeoCruncher is managed in a Git repository. To access the repository, you can use the Git command line, or alternatively a UI such as [SourceTree](http://sourcetreeapp.com).
+* Repository URL: https://github.com/ISSKA/geocruncher
+* Check out the **develop** branch before continuing.
+
+## Requirements
+
+To run geoCruncher need other libraries to be installed. The followings needs to be installed before running the python scripts:
+
+* Eigen
+* GmLib by brgm (Commit 2c7736a57906c48a8edfbc6348f0477512debbd6 or later)
+* MeshTools by brgm (Commit 20e68f3bcc0581cbc79affb6abc7b470fe0243fb or later)
+* scikit-image >= 0.14.0
+* CGAL >= 4.13
+* numpy
+ 
+##  Setup with linux 
+
+### Geocruncher
+
+1. run the following key to clone the geocruncher repository:
+
+        git clone https://github.com/ISSKA/geocruncher.git
+
+2. To be able to run the python script you need to manually change some files within geoCruncher. 
+
+	Within __Main__.py change:
+
+        from . import main
+
+	to:
+
+        import main
+
+	and within Main.py change:
+
+        from topography_reader import txt_extract
+        from ComputeIntersections import CrossSectionIntersections, MapIntersections, GeocruncherJsonEncoder, Slice
+        from MeshGeneration import generate_volumes
+
+	to:
+
+        from .topography_reader import txt_extract
+        from .ComputeIntersections import CrossSectionIntersections, MapIntersections, GeocruncherJsonEncoder, Slice
+        from .MeshGeneration import generate_volumes
+
+	To run locally the . before imports needs to be deleted.
```

Comment:
What does "run locally" mean? There is a difference between import GeoCruncher as a module and running the main script "directly". We shouldn't have to change the source code to use it within VK

Reply:
Then we have done something wrong because the geoCruncher code as is, just by cloning the gitHub and filling both:
```
   geocruncher-script-path 
    python-path
```
Simply do not run as a python script for me. Clark had to do the same changes as well. 
Maybe we have a different version of python but even just by running geocruncher directly in a python compiler it does fail.

And by locally I want to say "within your computer" as opposed to the VisualKarsys online webpage (but the term might be miss-leading)

Reply:
I have the same issue with the "dot imports" & was wondering what was correct

Reply:
I do not know what is correct but Ithis is relative imports: https://www.python.org/dev/peps/pep-0328/

Maybe we do not have the package but then it would need to be specified, because by just downloading the last python version it fails.

Reply:
Yes I think Philipp & I will have to fix this up. For me, GeoCruncher works from KARSYS *without* using the relative imports. (But still has an issue with the temp directory)